File: Code/HVIT/HVIT_JAVA/src/JavaPractice/JavaOOPBasic/LopHoc/LopHoc.java
package JavaPractice.JavaOOPBasic.LopHoc;

import java.util.Scanner;

public class LopHoc {
    private int MaLop;
    private String TenLop;
    private int SiSo;
    private String DiaChi;
    private String GVCN;

    public int getMaLop() {
        return MaLop;
    }

    public void setMaLop(int maLop) {
        MaLop = maLop;
    }

    public String getTenLop() {
        return TenLop;
    }

    public void setTenLop(String tenLop) {
        TenLop = tenLop;
    }

    public int getSiSo() {
        return SiSo;
    }

    public void setSiSo(int siSo) {
        SiSo = siSo;
    }

    public String getDiaChi() {
        return DiaChi;
    }

    public void setDiaChi(String diaChi) {
        DiaChi = diaChi;
    }

    public String getGVCN() {
        return GVCN;
    }

    public void setGVCN(String GVCN) {
        this.GVCN = GVCN;
    }

    public LopHoc() {
        Scanner sc = new Scanner(System.in);
        System.out.printf("Ma lop hoc: ");
        setMaLop(sc.nextInt());
        System.out.printf("Ten lop: ");
        sc.nextLine();
        setTenLop(sc.nextLine());
        System.out.printf("Si so: ");
        setSiSo(Integer.parseInt(sc.nextLine()));
//        sc.nextLine();
        System.out.printf("Dia chi: ");
        setDiaChi(sc.nextLine());
        System.out.printf("Giao vien chu nhiem: ");
        setGVCN(sc.nextLine());
    }

    public LopHoc(int maLop, String tenLop, int siSo, String diaChi, String gvcn) {
        setMaLop(maLop);
        setTenLop(tenLop);
        setSiSo(siSo);
        setDiaChi(diaChi);
        setGVCN(gvcn);
    }

    public void InThongTin() {
        System.out.println("Lop " + getTenLop() + " co ma la " + getMaLop() + " o dia chi " + getDiaChi() + "\n" +
                "Lop co " + getSiSo() + " hoc sinh va co GVCN la " + getGVCN());

    }
}

File: Code/HVIT/HVIT_EX/HVIT_OOP_EX/HVIT_OOP_EX/LopHoc.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HVIT_OOP_EX
{
    class LopHoc
    {
        public int MaLop { get; set; }
        public string TenLop { get; set; }
        public int SiSo { get; set; }
        public string DiaChi { get; set; }
        public string GiaoVienChuNhiem { get; set; }

        public LopHoc()
        {
            NhapThongTin();
        }

        public LopHoc(int MaLop, string TenLop, int SiSo, string DiaChi, string GiaoVienChuNhiem)
        {
            this.MaLop = MaLop;
            this.SiSo = SiSo;
            this.TenLop = TenLop;
            this.DiaChi = DiaChi;
            this.GiaoVienChuNhiem = GiaoVienChuNhiem;
        }

        public void NhapThongTin()
        {
            Console.Write("Ma lop: ");
            MaLop = int.Parse(Console.ReadLine());
            Console.Write("Ten lop: ");
            TenLop = Console.ReadLine();
            Console.Write("Si so: ");
            SiSo = int.Parse(Console.ReadLine());
            Console.Write("Dia chi: ");
            DiaChi = Console.ReadLine();
            Console.Write("Giao vien chu nhiem: ");
            GiaoVienChuNhiem = Console.ReadLine();
            Console.WriteLine();
        }

        public void HienThi()
        {
            Console.WriteLine($"Lop {TenLop} co ma la {MaLop} o {DiaChi}.");
            Console.WriteLine($"Lop co {SiSo} hoc sinh va co GVCN la {GiaoVienChuNhiem}.");
        }
    }
}
